src/main/java/seedu/application/logic/commands/CommandResult.java

```diff
@@ -18,22 +19,24 @@ public class CommandResult {
 
     /** The application should exit. */
     private final boolean exit;
+    private final Index interview;
 
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
+    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, Index interview) {
```

Comment:
Good modification of CommandResult class to achieve update to GUI. Let's see if there is any bugs